### ISSUE TITLE
Autorun at startup?

### ISSUE THREAD

**Issue by techmouse** [OatmealDome/SwapDRC#9]
Forgive my ignorance on this one as I'm still fairly new to Wii U homebrew. But is there a way to autorun this? Or maybe have a forwarder on the home menu?

The biggest reason I use this is to play Splatoon when in other rooms. I honestly feel like I don't play Splatoon nearly as much as I should, like the whole thing was a big waste of money, and it's because I hate monkeying with the TV or being restricted to the area right in front of the TV. And to play Splatoon when in another room, I would have to turn on the Wii U, run Haxachi, run the homebrew launcher, find SwapDRC in the list, run SwapDRC, and then I can run Splatoon. It seems like several steps could be skipped there and it feels as restrictive as playing on the TV.

I really hope I don't seem ungrateful because this is a great program. The swap is very smooth and it fixes a massive oversight most developers either didn't think of, or didn't care enough about. SwapDRC is a must have for any hacked Wii U and should be included in the default homebrew starter kit.

**Comment by CreeperMario** [comment]
Actually, you could configure Haxchi to directly launch SwapDRC.
The Haxchi installer reads a file on your SD Card during installation, `sd:/haxchi/config.txt`, and by changing this file, you can configure Haxchi to launch different programs when different buttons are held down.

Here's an example configuration:
```
a=wiiu/apps/mocha/mocha.elf
b=sysmenu
x=wiiu/apps/swapdrc/swapdrc.elf
default=wiiu/apps/homebrew_launcher/homebrew_launcher.elf
```
In this configuration, holding down the A button during Haxchi boot will make it launch Mocha, holding down the B button will make it launch its mini CFW, holding X will make it launch SwapDRC, and if none of those buttons are pressed, it will open the Homebrew Launcher.

Simply put this file on your SD Card and re-run the Haxchi installer.

**Comment by OatmealDome** [comment]
Thank you, @CreeperMario.

**Comment by techmouse** [comment]
@CreeperMario Now _that_ is some useful information. Does holding the button also work with the quick start menu? And you  mean to re-run the Haxchi installer in the homebrew launcher right?

**Comment by CreeperMario** [comment]
> Does holding the button also work with the quick start menu?

I don't think so, if you mean launching Haxchi from the Quick Start Menu, probably. You need to hold buttons down while Haxchi is booting.

> And you mean to re-run the Haxchi installer in the homebrew launcher right?

Yes.
